src/preact/base-element.js

```diff
@@ -395,6 +395,21 @@ export class PreactBaseElement extends AMP.BaseElement {
     });
   }
 
+  /** @override */
+  attemptChangeSize(newHeight, newWidth, opt_event) {
+    return super
+      .attemptChangeSize(newHeight, newWidth, undefined, opt_event)
+      .catch((e) => {
+        if (this.getOverflowElement && !this.getOverflowElement()) {
+          console./* OK */ warn(
```

Comment:
This might be an amp-specific feature, which would have to go on the amp-element.

Reply:
Good point. This is also done in `attemptyChangeHeight` method above, which is used other components. Created #35760 to tackle this later.

Reply:
Removed `attemptChangeSize` method & moved the check for overflow element in amp-iframe.